package.json

```diff
@@ -38,18 +41,32 @@
     "webfontloader": "^1.6.28"
   },
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@babel/plugin-transform-runtime": "^7.1.0",
-    "babel-eslint": "^10.0.1",
-    "babel-plugin-transform-class-properties": "^6.24.1",
+    "@types/cors": "^2.8.4",
+    "@types/express": "^4.16.1",
+    "@types/i18next": "^11.9.3",
```

Comment:
make sure the version of types is aligned to the package version, for instance types for `i18next` are v11.9.3 while the package is v9.0.10

Reply:
Unfortunately, not all the types libraries can be aligned to the libraries package versions we are using. I am going to update the types libraries to the closest versions, but still there will be differences for the following:

Library | Version | Closest types version
------------- | --------------- | --------------------------
i18next | ^14.1.1 | [12.1.0](https://www.npmjs.com/package/@types/i18next)
lodash | ^4.17.10 | [4.14.123](https://www.npmjs.com/package/@types/lodash)
papaparse | ^4.6.0 | [4.5.9](https://www.npmjs.com/package/@types/papaparse)
pouchdb | ^7.0.0" | [6.3.3](https://www.npmjs.com/package/@types/pouchdb)
pouchdb-browser | ^7.0.0" | [6.1.3](https://www.npmjs.com/package/@types/pouchdb-browser)
pouchdb-find | ^7.0.0" | [6.3.4](https://www.npmjs.com/package/@types/pouchdb-find)